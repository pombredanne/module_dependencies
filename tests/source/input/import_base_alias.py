import nltk as nlp

output = nlp.TextTilingTokenizer().tokenize(nlp.brown.raw()[0:10000])
