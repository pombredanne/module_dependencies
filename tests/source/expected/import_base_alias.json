{
    "dependencies": [
        "nltk.TextTilingTokenizer",
        "nltk.brown.raw"
    ],
    "imports": [
        "nltk"
    ]
}
